Rakefile

```diff
@@ -162,7 +162,9 @@ namespace :test do
       sh("cd #{integration_dir}/build/ && terraform workspace new #{tf_workspace}")
 
       # Generate Azure crendentials
-      creds = Train.create('azure').connection.connect
+      connection = Train.create('azure').connection
+      connection.connect
```

Comment:
That shouldn't be necessary? https://github.com/chef/train/blob/a931a4dccf8d01b0a99de28d5c17b2b484e6ce06/lib/train/transports/azure.rb#L50

Reply:
Ahh nice catch. I'll update.

Reply:
I updated and confirmed it works without the explicit connect.